Display MIT license correctly

lambdakilo opened the pull request.

akirataguchi115 committed:
Modify copyright information in LICENSE-CODE

Updated copyright year and holder in LICENSE-CODE.

akirataguchi115 committed:
Change license file reference in README

Updated license file reference from LICENSE to LICENSE-CODE.

ralyodio commented:
Just call it LICENSE -- why LICENSE-CODE?

akirataguchi115 commented:
Good question! LICENSE-CODE is a convention where we can separate the licensing of source code and other creative works. The MIT license is not really suited for non-code but it's applicable to code. For creative works Creative Commons are applicable licenses.

You can see how GitHub uses the aforementioned convention in their repos: https://github.com/github/docs/blob/main/LICENSE-CODE

akirataguchi115 commented:
sure i can add cc0 if you specifically want the most permissive non-code license paired with the mit source code license!

akirataguchi115 committed:
Add CC0 1.0 Universal license

Added the CC0 1.0 Universal license to the project, allowing for public domain dedication of the work.
